experimental/dds/tree/src/EditLog.ts

```diff
@@ -405,7 +487,10 @@ export class EditLog<TChange = unknown> extends TypedEventEmitter<IEditLogEvents
 
 				editChunk.edits = edits;
```

Comment:
Should we be assigning the .edits field in the case where we aren't adding it to loadedChunkCache? Would it ever get evicted (set to null) in that case? I think we'd retain it forever.

Reply:
(not that this situation should ever happen, since history and eviction are mutually exclusive)

Reply:
The original porpuse of loadedChunkCache was to only cache and evict(even thought the size of it right now is Infinity that prevents eviction but that could change in the future) edits that are not in the current session because getEditInSessionAtIndex is synchronous and needs the edits in memory(this could change if we make getEditInSessionAtIndex and tryGetEditInSessionAtIndex asynchronous), so in this case history and this kind of eviction(that is not the one I'm implementing) are not mutually exclusive